Traceback (most recent call last):
  File "/home/emilio/Documents/ai/pizza/scripts/train_pizza_rl.py", line 28, in <module>
    import wandb
ModuleNotFoundError: No module named 'wandb'
